### PR TITLE
Add TabSizeGet and TabSizeSet APIs

### PR DESCRIPTION
**Pull request by dagon666**
TabSizeGet and TabSizeSet is a pair of APIs allowing the modification of
the ncurses' global variable TABSIZE. TABSIZE defines the tab width of
the \t character.

### CODE REVIEW

File: ncurses.go [hunk 1]
```diff
@@ -307,3 +307,14 @@ func UseDefaultColors() error {
 func UseEnvironment(use bool) {
 	C.use_env(C.bool(use))
 }
+
+// TabSizeSet allows for modification of the tab width. This setting is global
+// and affects all windows.
+func TabSizeSet(tabSize int) {
```
Comment (rthornton128) [inline ncurses.go]: To match the project style, this should be renamed to:

```suggestion
func SetTabSize(tabSize int) {
```

File: ncurses.go [hunk 2]
```diff
@@ -307,3 +307,14 @@ func UseDefaultColors() error {
...
+	C.TABSIZE = C.int(tabSize)
+}
+
+// TabSizeSet returns the configured tab width.
+func TabSizeGet() int {
```
Comment (rthornton128) [inline ncurses.go]: To match the project style, this should be renamed to:
```suggestion
func TabSize() int {
```

File: ncurses.go (hunk 1 above)
Comment (dagon666) [reply ncurses.go]: Updated accordingly. Thanks for reviewing.

File: ncurses.go (hunk 2 above)
Comment (dagon666) [reply ncurses.go]: Updated accordingly. Thanks for reviewing.

**Pull request merged**